developer-docs-site/docs/guides/prover-guides/spec-lang.md

```diff
@@ -705,31 +620,30 @@ spec get_one_off {
 
 > TODO: `[check]` property is currently not longer implemented
```

Comment:
Is this TODO still relevant? Can we move to an issue for tracking instead?

Reply:
Done.